pkg/virt-api/rest/subresource_test.go

```diff
@@ -307,18 +307,6 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 			close(done)
 		}, 5)
 
-		It("should fail to connect to VNC if the VMI is paused", func(done Done) {
-
-			request.PathParameters()["name"] = "testvmi"
-			request.PathParameters()["namespace"] = "default"
-
-			expectVMI(true, true)
-
-			app.VNCRequestHandler(request, response)
-			ExpectStatusErrorWithCode(recorder, http.StatusConflict)
```

Comment:
Maybe it's better to keep the test and to expect that the the connection will succeed? Exactly like you did for functional tests.